examples/platforms/nrf52811/README.md

```diff
@@ -123,27 +163,31 @@ Now you are connected with the CLI, commands below shoud be used to form a netwo
  Done
  ```
 
-4. After a couple of seconds the node will become a Leader of the network.
+After a couple of seconds the node will become a Leader of the network.
 
  ```bash
  > state
  Leader
  ```
 
-2. The CLI MTD example uses the direct UART connection. To view raw UART output, start a terminal
+8. Open a terminal connection on the second board and attach a node to the network.
+   The CLI MTD example uses the direct UART connection. To view raw UART output, start a terminal
    emulator like screen and connect to the used COM port with the following UART settings:
     - Baud rate: 115200
     - 8 data bits
     - 1 stop bit
     - No parity
     - HW flow control: RTS/CTS
-   Open a terminal connection on the second board and attach a node to the network.
+
+On Linux system, run the following to connect to the second board.
```

Comment:
Is this an Linux version of the step 8? If so, maybe make it a note? It looks a bit weird here like this.

Reply:
fixed